packages/strapi-plugin-content-manager/admin/src/components/FieldComponent/index.js

```diff
@@ -20,6 +20,7 @@ const FieldComponent = ({
   icon,
   isFromDynamicZone,
   isRepeatable,
+  isSub,
```

Comment:
`isSub` is not very clear for me. I would change this property name by something like `isRecursive` or `isNested`.